test/built-ins/Math/sumPrecise/prop-desc.js

```diff
@@ -0,0 +1,14 @@
+// Copyright (C) 2024 Kevin Gibbons. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: sec-math.sumprecise
+description: checks the "sumPrecise" property of Math
+includes: [propertyHelper.js]
+features: [Math.sumPrecise]
+---*/
+
+verifyProperty(Math, "sumPrecise", {
+  writable: false,
+  enumerable: false,
+  configurable: true
+});
```

Comment:
I don't think this is correct:

> Every other [data property](https://tc39.es/ecma262/#sec-object-type) described in clauses [19](https://tc39.es/ecma262/#sec-global-object) through [28](https://tc39.es/ecma262/#sec-reflection) and in Annex [B.2](https://tc39.es/ecma262/#sec-additional-built-in-properties) has the attributes { [[Writable]]: true, [[Enumerable]]: false, [[Configurable]]: true } unless otherwise specified.

Reply:
good catch; my tests check enumerability and configurability but not writability.